.github/workflows/deploy-mainnet.yaml

```diff
@@ -104,13 +105,32 @@ jobs:
           HAPI_EOSRATE_GET_STATS_URL: ${{ secrets.HAPI_EOSRATE_GET_STATS_URL }}
           HAPI_EOSRATE_GET_STATS_USER: ${{ secrets.HAPI_EOSRATE_GET_STATS_USER }}
           HAPI_EOSRATE_GET_STATS_PASSWORD: ${{ secrets.HAPI_EOSRATE_GET_STATS_PASSWORD }}
-
+          # hapi-evm
+          HAPI_EVM_SERVER_PORT: '9090'
+          HAPI_EVM_SERVER_ADDRESS: '0.0.0.0'
+          HAPI_EVM_HASURA_URL: 'http://dashboard-hasura:8080/v1/graphql'
+          HAPI_EVM_HASURA_ADMIN_SECRET: ${{ secrets.HAPI_EVM_HASURA_ADMIN_SECRET }}
+          HAPI_EVM_DATABASE_URL: ${{ secrets.HAPI_EVM_DATABASE_URL }}
```

Comment:
@leisterfrancisco we will need to set this secret in github, can you send the value needed over slack please